projects/self-creation-portal/src/app/services/form/form.service.ts

```diff
@@ -89,7 +90,36 @@ export class FormService {
         reject(error);
       }
     });
-  
-}
 
+  }
+
+  private createUrlWithParams(baseUrl: string, params: { [key: string]: any }): string {
```

Comment:
this can be used for other places as well right, then why it's in form service ?